### PR TITLE
[Discounts] fix: show accurate subtotal price for cart

### PR DESCRIPTION
**Pull request by mathiusj**
### PR Summary: 

Updates total price to be items_subtotal_price to more accurately represent the value after line item discounts only.

Before:

image

After:

image

image

### Why are these changes introduced?

fixes: https://github.com/Shopify/core-issues/issues/57440
fixes: https://github.com/Shopify/dawn/issues/2781

### Decision log

| # | Decision | Alternatives | Rationale | Downsides |
|---|---|---|---|---|
| 1 |   |   |   |   |

### Testing steps/scenarios

manually edited in store editor: https://admin.web.fix-inaccurate-total.mathius-johnson.us.spin.dev/store/shop1/themes/1?key=sections%2Fmain-cart-footer.liquid

cart: https://shop1.shopify.fix-inaccurate-total.mathius-johnson.us.spin.dev/cart

- add products to cart
- go to checkout and add product + order discounts (take note of total price after order discount)
- go back to cart and confirm subtotal accurately reflects amount of line items - line item discounts, order discount not included (should be a higher amount than total price with order discount applied)

### Demo links

- [Store](https://shop1.shopify.fix-inaccurate-total.mathius-johnson.us.spin.dev/cart)
- [Editor](https://admin.web.fix-inaccurate-total.mathius-johnson.us.spin.dev/store/shop1/themes/1?key=sections%2Fmain-cart-footer.liquid)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Comment by devisscher** [comment]
This looks clearer for the merchant, but as you mentioned @mathiusj, an additional total line after the order discounts are applied could make it even clearer. Another thing to account for is the cart drawer, which should be found [here](https://admin.web.fix-inaccurate-total.mathius-johnson.us.spin.dev/store/shop1/themes/1?key=snippets%2Fcart-drawer.liquid), see screenshot below.: 

image

File: assets/component-totals.css [hunk 1]
```diff
@@ -21,7 +21,11 @@
   margin-left: 2rem;
 }
 
-.totals__subtotal-value {
+.totals__total {
+  margin-top: .5rem;
+}
```
Comment (ludoboludo) [inline assets/component-totals.css:26]: I don't think we need this. It applies that margin top even when there are no discounts above it. 

We do have this existing declaration: 
```css
.cart__footer .discounts {
    margin-top: 1rem;
}
```

Which could be changed to `margin-bottom` 👍

Comment (ludoboludo) [reply assets/component-totals.css:26]: Then I'd like to see what a designer will think as the spacing between the total and the `taxes and shipping...` is bigger than between the discount and the total 🤔

**Review by ludoboludo — APPROVED** [review]
Looking good to me 👍 

One note, some stores could be using free shipping methods only and include taxes in their product prices. So the total amount on the cart page could be exactly the same as the one during checkout. 
Though since you can always add a discount during checkout, I think it's ok to keep it as is 👍

File: locales/en.default.json [hunk 2]
```diff
@@ -272,8 +272,8 @@
       "title": "Your cart",
       "caption": "Cart items",
       "remove_title": "Remove {{ title }}",
-      "subtotal": "Subtotal",
-      "new_subtotal": "New subtotal",
+      "estimated_total": "Estimated total",
```
[the remark this answers is not in the document]
Comment (mathiusj) [reply locales/en.default.json:275]: Thank you!!! 🙏

**Review by devisscher — APPROVED** [review]

**Review by kjellr — APPROVED** [review]
Looks good from the UX side. Thank you! 🙌

**Pull request merged**